contrib/get-gmp

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# This script should only be used if your distribution does not ship with the
+# GMP configuration you need. For example, contrib/get-win-dependencies
+# cross-compiles GMP for Windows. You can also use the script if your
+# distribution does not ship with static GMP libraries (e.g., Arch Linux) and
+# you want to build CVC4 statically.
+# In most of the cases the GMP version installed on your system is the one you
+# want and should use.
+#
+
+source "$(dirname "$0")/get-script-header.sh"
+
+if [ -z "${BUILD_TYPE}" ]; then
+  BUILD_TYPE="--disable-shared --enable-static"
```

Comment:
Nit: why use a different pattern than for `$HOST` and `$GMPVERSION` to set the default here? :)

Reply:
Because by default we want a static GMP library if we only call contrib/get-cmp?